src/Modules/CurrentMapModule/Templates/CurrentMapWidget.mt

```diff
@@ -0,0 +1,15 @@
+﻿<component>
+    <using namespace="EvoSC.Common.Interfaces.Models"/>
+    <property type="IMap" name="map"/>
+    <property type="string" name="country"/>
+    <template>
+        <frame pos="158 88">
+            <label pos="-11 -3" z-index="0" size="60 5" text="$s{{map.Name}}" textfont="GameFontSemiBold" halign="right"
+                   valign="center2" textsize="3"/>
+            <label id="" pos="-11 -7.5" z-index="0" size="60 5" text="$s{{map.Author.NickName}}" halign="right"
+                   textfont="GameFontRegular" valign="center2" textsize="2"/>
+            <quad pos="0 -5" image="file://Media/Flags/{{country}}.dds" z-index="0" size="10 10" bgcolor="0000" opacity="1"
+                  halign="right" valign="center" keepratio="Fit" />
+        </frame>
+    </template>
+</component>
```

Comment:
We should decide whether we want to use the native elements or use the wrappers provided by the template engine. 
@brakerb, @snixtho

Reply:
Wrappers can be handy when you need automate xy coordinates, though i wish to use native elements when possible, since it's easier to generate the xml using the interface designer.

Reply:
If by wrappers, you mean components exposed by EvoSC so that we can have a consistent layout and style across the entire UI, then yes I agree we should do that instead of native elements. But for that we first need a design, and I'm currently in discussion with nj for that.